docs/source/gbd2017_models/risk_attributable_causes/diabetes_mellitus/index.rst

```diff
@@ -81,9 +81,13 @@ This cause model is designed to simulate the basic structure of the risk factor
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
-1. Lack of remission data in GBD 2017: The assumptions made for this cause model include that there are no remission rates found for overall diabetes mellitus, diabetes mellitus type 1, or diabetes mellitus type 2. The limitations of these assumptions impose a lack of remission and accuracy on the applicability of the model. According to external sources and GBD 2017, diabetes mellitus and its subcauses should include remission rates but there is no data for remission found in GBD 2017. 
+1. We are using remission data from dismod, modelable_entity_id_2005 for overall DM. 
 
-2. Case definition cross-walks on FPG and HbA1c: GBD 2017 assumed that HbA1c >6.5% was equivalent to FPG >126 mg/dL.
+2. In vivarium, 'uncomplicated DM Type 1 and Type 2' is 'Moderate', which is different from how GBD 2017 is modelling it. In the future, severity splits will be revisited using disability weights. 
+
+3. DM is non-fatal only. CSMR is only applicable to Severe DM.
```

Comment:
I don't understand this.  People will die of DM in the sim, right?

Reply:
yes.  maybe the text above should say "EMR is greater than zero for severe diabetes only".  is CSMR the right mortality, or should it be EMR?
anyway, EMR = 0 for mild/moderate sequelae.